pom.xml

```diff
@@ -223,6 +223,11 @@
         <version>1.1.1</version>
       </dependency>
       <dependency>
+        <groupId>commons-pool</groupId>
+        <artifactId>commons-pool</artifactId>
+        <version>1.6</version>
```

Comment:
Yes, I think we should. For a new dependency, I think we should be using the latest available, to ensure the best upstream support.